webapp/src/main/webapp/templates/freemarker/body/roles/userRoles-list.ftl

```diff
@@ -0,0 +1,42 @@
+<h2>User roles</h2>
+
+<section id="show-auth" role="region">
+	<#assign rolesControllerUrl = urls.base + "/admin/roles" >
+	<script type="text/javascript">
+		function editRole(roleUri, roleLabel){
+		  let newLabel = prompt("${i18n().role_rename_prompt?js_string}", roleLabel);
+		  if (newLabel && !(/^\s*$/.test(newLabel))) {
+		    let form = document.forms["edit-" + roleUri];
+		    form.label.value = newLabel;
+		    form.submit();
+		  }
+		}
```

Comment:
I don't like the approach for editing role names based on a prompt box. Any chance to make a small form for this? I think we usually used this approach in the VIVO application.

Reply:
Entry forms work well for data stored in content, not so much for configuration graphs.